### PR TITLE
Полная разметка главной страницы

### PR DESCRIPTION
**Pull request by alex21072**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,107 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
+    <header class="main-header">
+      <a href="#"><img src="#" height="100" width="100" alt="Логотип"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 2]
```diff
@@ -5,8 +5,107 @@
...
+            <a href="#">Главная</a>
+          </li>
+          <li class="site-navigation-item">
+            <a href="#">О Седоне</a>
+          </li>
+          <li class="site-navigation-item">
+            <a href="#">Гостинницы</a>
+          </li>
+        </ul>
+        <a href="#"><img src="#" width="18" height="18" alt="Поиск"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 3]
```diff
@@ -5,8 +5,107 @@
...
+        <a href="#"><img src="#" width="18" height="18" alt="Избранное"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 4]
```diff
@@ -5,8 +5,107 @@
...
+        <button>Хочу сюда!</button>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <img src="#" width="188" height="45" alt="welcome">
+        <img src="#" width="256" height="29" alt="to the garden">
+        <img src="#" width="457" height="213" alt="SEDONA becouse the grand canyon sucks!">
+        <h1>Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
```
Comment (prikhodkin) [inline index.html]: Don't use br tag

File: index.html [hunk 5]
```diff
@@ -5,8 +5,107 @@
...
+        <a href="#"><img src="#" width="18" height="18" alt="Поиск"></a>
+        <a href="#"><img src="#" width="18" height="18" alt="Избранное"></a>
+        <button>Хочу сюда!</button>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <img src="#" width="188" height="45" alt="welcome">
+        <img src="#" width="256" height="29" alt="to the garden">
+        <img src="#" width="457" height="213" alt="SEDONA becouse the grand canyon sucks!">
```
Comment (prikhodkin) [inline index.html:37]: Please merge 3 pictures into 1

File: index.html [hunk 6]
```diff
@@ -5,8 +5,107 @@
...
+              <p class="advantages-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+            </li>
+            <li class="advantages-item">
+              <h2>Мало туристов</h2>
+              <p class="advantages-description">№5</p>
+              <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+            </li>
+        </ul>
+        <img src="№" width="800" height="384" alt="Мост дьявола">
+      </section>
```
Comment (prikhodkin) [inline index.html]: Not valid code. Please check the nesting of tags. ( ul > li )

File: index.html [hunk 7]
```diff
@@ -5,8 +5,107 @@
...
+        <h3>Подпишитесь на рассылку</h3>
+        <p>Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <input class="malling-input" type="text" value="Ваш e-mail">
+        <button>ПОДПИСАТЬСЯ</button>
+      </section>
+    </main>
+    <footer class="main-footer">
+      <ul class="social-list">
+        <li class="social-list-item">
+          <a href="#"><img src="#" width="70" height="30" alt="Twitter"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 8]
```diff
@@ -5,8 +5,107 @@
...
+        </li>
+        <li class="social-list-item">
+          <a href="#"><img src="#" width="70" height="30" alt="Facebook"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 9]
```diff
@@ -5,8 +5,107 @@
...
+        </li>
+        <li class="social-list-item">
+          <a href="#"><img src="#" width="70" height="30" alt="YOutube"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 10]
```diff
@@ -5,8 +5,107 @@
...
+        </li>
+      </ul>
+      <div class=phone><a href="tel:+78128121212">+7 (812) 812-12-12</a>
+      </div>
+      <a href="#"><img src="#" width="115" height="34" alt="Логотип HTML"></a>
```
Comment (prikhodkin) [inline index.html]: Please move img tag on the next line

File: index.html [hunk 11]
```diff
@@ -5,8 +5,107 @@
...
+          <a href="#"><img src="#" width="70" height="30" alt="Twitter"></a>
+        </li>
+        <li class="social-list-item">
+          <a href="#"><img src="#" width="70" height="30" alt="Facebook"></a>
+        </li>
+        <li class="social-list-item">
+          <a href="#"><img src="#" width="70" height="30" alt="YOutube"></a>
+        </li>
+      </ul>
+      <div class=phone><a href="tel:+78128121212">+7 (812) 812-12-12</a>
```
Comment (prikhodkin) [inline index.html]: div is redundant. Use just <a href="tel:+78128121212">+7 (812) 812-12-12</a>

**Review by prikhodkin — CHANGES_REQUESTED** [review]
Please check issues

**Comment by alex21072** [comment]
corrected all the comments

**Review by prikhodkin — APPROVED** [review]

**Pull request merged**